### ISSUE TITLE
More Information

### ISSUE THREAD

**Issue by Kleptine** [Leandros/p4fs#1]
I'm pretty interested in your use case / implementation as I'm in the process of figuring out better source control systems for games. Is there any more information on this project?

**Comment by Leandros** [comment]
This was done as a rather speedy prototype, to try if I could map a perforce depot to a filesystem. Here is a little gif demonstrating it in action: https://twitter.com/ArvidGerstmann/status/1020671795425357830

At this stage, it doesn't do much more (I only spend about one evening on this). To add syncing and checking out files, you'd probably have an actual perforce workspace setup, and than add this on top of it. 

It's also currently only running on macOS.

**Comment by Kleptine** [comment]
Ahh, gotcha. Pretty cool, though!

Have you taken a poke at GVFS? It does a similar thing for Git.

**Comment by Leandros** [comment]
Yes, this was more or less inspired by GVFS and some conversation I had on Twitter. 
It's significantly more complicated to build GVFS in Git, than to do something similar in Perforce (or SVN, anything which requires a constant server connection). I think I've proven that successfully. ;)
